export.rb

```diff
@@ -1,108 +1,85 @@
 require 'csv'
 
 class Przejazd
-  attr_accessor :numer_pojazdu, :data_wyjazdu, :data_zjazdu, :km_wyjazd, :km_powrot, :paliwo_wyjazd, :paliwo_dodatkowo, :paliwo_zjazd, :zuzycie, :przejechane_kilometry, :norma
 
-    def zbierz_dane
-    puts "Podaj numer rejestracyjny pojazdu"
-    @numer_pojazdu = gets.chomp.upcase
-   
-    puts "Podaj datę wyjazdu"
-    @data_wyjazdu = gets.chomp
-   
-    puts "Podaj datę zjazdu"
-    @data_zjazdu = gets.chomp
-   
-    puts "Podaj stan licznika podczas wyjazdu w trasę"
-    @km_wyjazd = gets.chomp.to_f
-   
-    puts "Podaj stan licznika po powrocie z trasy"
-    @km_powrot = gets.chomp.to_f
-   
-    puts "Podaj stan paliwa przy wyjeździe w trasę"
-    @paliwo_wyjazd = gets.chomp.to_f
-   
-    puts "Czy podczas wyjazdu było dotankowywane paliwo? Podaj: TAK/NIE"
-    @paliwo_dodatkowo = 0
-     until @tankowania_w_trasie == "NIE"
-      @tankowania_w_trasie = gets.chomp.upcase
-       if @tankowania_w_trasie == "TAK"
-        puts "Podaj wartość zatankowanego paliwa"
-        @paliwo_dodatkowo = @paliwo_dodatkowo + gets.chomp.to_f
-        puts "Czy było kolejne tankowanie? Podaj: TAK/NIE"
-       end
-     end
-   
-    puts "Podaj stan paliwa po powrocie z trasy"
-    @paliwo_zjazd = gets.chomp.to_f
-    end
-   
-    def norma_spalania
-    @zuzycie = @paliwo_wyjazd + @paliwo_dodatkowo - @paliwo_zjazd
-    @przejechane_kilometry = @km_powrot - @km_wyjazd
-    @norma = (@zuzycie/@przejechane_kilometry)*100
-    puts "Twoja norma to #{@norma}"
-    end
+ def initialize(numer_pojazdu, data_wyjazdu, data_zjazdu, km_wyjazd, km_powrot, paliwo_wyjazd, paliwo_dodatkowo, paliwo_zjazd, zuzycie, przejechane_kilometry, norma, raport_path)
 
-def zapisz
-raport_path = "files/#{@numer_pojazdu}.csv"
+  @numer_pojazdu = numer_pojazdu
+  @data_wyjazdu = data_wyjazdu
+  @data_zjazdu = data_zjazdu
+  @km_wyjazd = km_wyjazd
+  @km_powrot = km_powrot
+  @paliwo_wyjazd = paliwo_wyjazd
+  @paliwo_dodatkowo = paliwo_dodatkowo
+  @paliwo_zjazd = paliwo_zjazd
+  @zuzycie = zuzycie
+  @przejechane_kilometry = przejechane_kilometry
+  @norma = norma
+  @raport_path = raport_path
+ end
 
- if File.exists?(raport_path)
-   CSV.open(raport_path, "a+") do |csv|
+ def norma_spalania
+  @zuzycie = @paliwo_wyjazd + @paliwo_dodatkowo - @paliwo_zjazd
+  @przejechane_kilometry = @km_powrot - @km_wyjazd
+  @norma = (@zuzycie/@przejechane_kilometry)*100
+  puts "Twoja norma to #{@norma}"
```

Comment:
Nie chcemy, żeby przejazd wiedział cokolwiek o terminalu. Dodatkowo opis tekstowy też tu jest dlatego, że wcześniej wypisywaliśmy to w terminalu. Jeżeli norma będzie wyświetlana na stronie www, to pewnie w HTML ten opis już będzie i tylko będziemy chceli uzupełnić wartość samej normy. Ta metoda powinna tylko zwrócić liczbę (obliczoną normę).
```
return @norma
```

Reply:
absolutna racja, podmieniłem na return

Reply:
👍